com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -135,8 +135,16 @@ public static int maxVisibleAdditionalLights
         private readonly DebugDisplaySettingsUI m_DebugDisplaySettingsUI = new DebugDisplaySettingsUI();
 #endif
 
+        private UniversalRenderPipelineGlobalSettings m_GlobalSettings;
+        public override RenderPipelineGlobalSettings defaultSettings => m_GlobalSettings;
+
         public UniversalRenderPipeline(UniversalRenderPipelineAsset asset)
         {
+#if UNITY_EDITOR
+            m_GlobalSettings = UniversalRenderPipelineGlobalSettings.Ensure();
+#else
+            m_GlobalSettings = UniversalRenderPipelineGlobalSettings.instance;
+#endif
```

Comment:
Do you see in future we could possibly call `Ensure` in other places? Maybe a safer approach is to also declare Ensure in standalone and mode this if UNITY_EDITOR there?